google-cloud-automl/lib/google-cloud-automl.rb

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/automl"
```

Comment:
We usually only add these files to gems with a hand-written layer. Are we expecting non-generated code?

Reply:
No we aren't expecting non-generated code. But shouldn't we have these files anyway? Bundler tries to load them by default.

Reply:
My question isn't about correctness, but about consistency. I am generally favorable to these files, but all of the other generated gems (other than container_analysis) omit this file. Is this file generated?

Reply:
It's not generated. I think it should be. (We should add it to the new generator.) It's been somewhere down on my list for a long time to add it manually to the other generated libraries. I should probably just file an issue. :)

Reply:
https://github.com/googleapis/gapic-generator-ruby/issues/181

Reply:
Also: https://github.com/googleapis/google-cloud-ruby/issues/3541